lib/formatter/formatterUtils.js

```diff
@@ -16,7 +19,25 @@ function wrapAsIdentifier(value, builder, client) {
   return client.wrapIdentifier((value || '').trim(), queryContext);
 }
 
+function formatDefault(value, type, client) {
+  if (value === void 0) {
+    return '';
+  } else if (value === null) {
+    return 'null';
+  } else if (value instanceof Raw) {
```

Comment:
You can use
```
if (value && value.isRawInstance) {
```
for doing the check without requiring Raw.

Reply:
Done, thanks!